model/src/test/kotlin/licenses/LicenseInfoResolverTest.kt

```diff
@@ -70,7 +70,7 @@ class LicenseInfoResolverTest : WordSpec() {
             type = VcsType.GIT,
             url = "https://github.com/oss-review-toolkit/ort.git",
             revision = "master",
-            resolvedRevision = "master"
+            resolvedRevision = "0000000011111111222222223333333344444444"
```

Comment:
Nit: I believe we usually used all zeros, see `git grep resolvedRevision.*0000`.

Reply:
The output of that grep command yields 3 matches for the zero case and 2 matches for the non-zero case.
Can we keep it as is?

Reply:
I won't block it if you keep it, but I'll probably do an alignment-commit afterwards.